VIDEO_5_SCRIPT.md

```diff
@@ -0,0 +1,71 @@
+
+## AsyncAPI and Linux Foundation
+
+At that still initial moment, the project receives a significant Postman sponsorship that enables several individuals to work full time on the project. It was then that discussions began on what was the best way to improve governance within the project. 
+
+If one thing was crystal clear, it was that AsyncAPI should join a neutral home. But not just any neutral home. It needed to join a well-established and trusted organization. Housing the project on such a solid and neutral ground would help to develop its independence and community-driven management. 
+
+From the beginning, AsyncAPI has the idea to democratize power and establish a horizontal model. Thus encouraging multiple points of view could enrich its vision, decision-making, and governance. 
+
+With these ideas and principles clear, after a long time of searches and negotiations, AsyncAPI joined the Linux Foundation umbrella in March 2021. This move granted neutrality and open governance, ensuring that the specification will continue to be driven democratically by the people who contribute their time and effort to the project.
+
+## What is transparency
+
+Organizational transparency can be defined as a deliberate attempt by the organization to create an environment of trust in which to promote free access to information, open communication, and the participation of all professionals in decision-making. 
+
+When we talk about transparency, we refer to that attitude and methodology that allows controlling the financial aspects of a company to fulfill its mission and vision with the participation of all the people in an organization. Transparency requires obligations, clarity, and implication to sustain it. The information provided must be truthful, based on evidence that can be checked and verified.
+
+Transparency has become a fundamental concept in the democratization processes undertaken by governments, institutions, companies, organizations, projects... in recent years. In its ideal state, democracy is nothing more than a method that allows the community members to analyze reality from different points of view to make decisions freely and for the common good.
+
+## What is open governance
+
+Open governance is a model of governance that promotes the principles of transparency, integrity, accountability, and stakeholder participation in support of democracy and inclusive growth. Driven by the participation of all the people that make up a project, company, or institution, it promotes engagement and democracy.
+
+In the same way, this model facilitates working with like-minded people living thousands of miles away to pursue and achieve a common goal. The intention is to break down individualism and hierarchical archetypes while upholding a system that gives autonomy and decision-making power to its members.
```

Comment:
*like-minded* conveys the idea of an ideological bubble. Instead of this, I would say something like *collaboration among people who might be located in distant regions but who share the same values*